src/MongoDB/ServerDescription.c

```diff
@@ -309,22 +296,27 @@ static zend_object* php_phongo_serverdescription_create_object(zend_class_entry*
 	return &intern->std;
 } /* }}} */
 
-static HashTable* php_phongo_serverdescription_get_debug_info(phongo_compat_object_handler_type* object, int* is_temp) /* {{{ */
+HashTable* php_phongo_serverdescription_get_properties_hash(phongo_compat_object_handler_type* object, bool is_debug) /* {{{ */
 {
-	php_phongo_serverdescription_t* intern = NULL;
-	zval                            retval = ZVAL_STATIC_INIT;
+	php_phongo_serverdescription_t* intern;
+	HashTable*                      props;
 
-	*is_temp = 1;
-	intern   = Z_OBJ_SERVERDESCRIPTION(PHONGO_COMPAT_GET_OBJ(object));
+	intern = Z_OBJ_SERVERDESCRIPTION(PHONGO_COMPAT_GET_OBJ(object));
+
+	PHONGO_GET_PROPERTY_HASH_INIT_PROPS(is_debug, intern, props, 0);
```

Comment:
For now, an empty object will do. We could expose the last hello response from the document as individual properties, but I would suggest raising this with @jmikola in the original PR.